### ISSUE TITLE
Proper location for .Rmd files when deploying with rsconnect

### ISSUE THREAD

**Issue by mcsiple** [ThinkR-open/golem#729]
Hi golem developers,
I am building my app with golem and would like to set up deployment so I can do it quickly after making changes to the code. Local deployment works just fine. When I tried to use the following:
```
golem::add_shinyserver_file()
rsconnect::deployApp()
```
to deploy the app to my shiny server, it worked perfectly except for a downloadable report that I have as part of the app. I get a "failed - Server problem" error when I try to download from the server, which to me indicates that it can't find the template file. 

Currently the `report.Rmd` template file is in `inst/report/report.Rmd`. 
In the `server.R` file, `report.Rmd` is copied from the `inst/` folder to a temp directory as follows:
```
tempReport <- file.path(tempdir(), "report.Rmd")
file.copy(from = app_sys("report","report.Rmd"), tempReport, overwrite = TRUE)
```
In other apps with the same feature (downloadable html report), the `report.Rmd` file is in the same directory as `app.R`. But I know for package purposes it should be in `inst/`. What is the answer? 

The package is [here](https://github.com/mcsiple/mmrefpoints). Thank you!

**Comment by statnmap** [comment]
Hi Margaret,

Your way is the way to include the report template.
It works locally, so it should work remotely.
The problem may be somewhere else. Can you share the log ? And describe the steps you are doing exactly ?

Note that I am not able to deploy your application to ShinyApps.io from your _master_ branch
I got the following error. It is probably a mismatch between your OS and my Linux. EOF is for End Of Line character, but I do not see which file is not compatible.
```
Erreur dans value[[3L]](cond) : parse error: premature EOF
                                       
                     (right here) ------^
Appels : local ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Exécution arrêtée
```

**Comment by teofiln** [comment]
I think I've seen a similar error with non-ascii characters. Running `devtools::check()` shows these files as having non-ascii. 
```
> checking R files for non-ASCII characters ... WARNING
  Found the following files with non-ASCII characters:
    00_setup.R
    13_pop_vs_yield.R
    15_plot_bycatch_guesses.R
    17_plot_proj.R
    18_multiplot_proj.R
    app_server.R
    app_ui.R
  Portable packages must use only ASCII characters in their R code,
  except perhaps in comments.
  Use \uxxxx escapes for other characters.
```

Might be worth taking a look.

**Comment by mcsiple** [comment]
Thank you @statnmap -- I fixed it! I was having a basic issues with dependencies (using a library in the report that wasn't in the NAMESPACE file) and discovered it by looking at the logs as you suggested. Rookie mistake. Now it works! 🎉  Thank you. Still not sure about that installation issue. It must be an issue with Linux specifically because I have been able to install and run it on Mac and Windows machines with no problem. 

@teofiln -- thank you for checking those too. I know there are non-ASCII characters in the package, and I have yet to find a good solution to this issue. It's not a golem issue; just an issue with automating translation for my figure legends. If you have specific advice about the translation/accents stuff, definitely add it [here](https://github.com/mcsiple/mmrefpoints/issues/52).
